### PR TITLE
adding visualization to rviz via nav_msgs/Path

### PR DESCRIPTION
**Pull request by ferdianjovan**
Hi @cdondrup, please test this in your environment. I just tested it in uol simulator and it worked fine.

### CODE REVIEW

File: human_trajectory/scripts/trajectory_publisher.py [hunk 1]
```diff
@@ -71,9 +77,31 @@ def _publish_online_data(self, incremental=False):
...
             self._publish_rate.sleep()
 
+    # add each traj to be published in nav_msg/Path
+    def _add_in_nav_msgs(self, uuid):
+        if uuid not in self.pub_nav:
+            rospy.loginfo("Creating a publisher for %s...", uuid)
+            name = uuid.replace("-", "0")
+            self.pub_nav[uuid] = rospy.Publisher(
+                self.name + '/' + name, Path, queue_size=10
+            )
```
Comment (cdondrup) [inline human_trajectory/scripts/trajectory_publisher.py:92]: Did you try making that a latched topic so even if the person is not tracked any more, we could still visualise it?

**Comment by cdondrup** [comment]
Looks great. Thank you!
I'll try it next week.

**Comment by ferdianjovan** [comment]
Actually, no. As long as the person is still detected you can visualize it. Once the person disappears, the visualization is gone.

Do you want me to do the otherway around?

**Comment by cdondrup** [comment]
If it works with latched topics, that would be great. Otherwise you have to fiddle with the rviz guy as long as the person is detected and cannot look at it afterwards. If you could just add a `latch=True` to the publisher and try if that works, that'd be great. Also so you would have to keep the publishers alive for some time after the detection was lost...

**Comment by ferdianjovan** [comment]
Ok, I just tested it twice in the simulator and it worked fine. So the publisher for each uuid will be alive 1 min after the detection was lost. Hope it will be ok now.

**Comment by cdondrup** [comment]
Thank you!

**Pull request merged**